src/fuse/channel.rs

```diff
@@ -30,7 +31,7 @@ fn with_fuse_args<T, F: FnOnce(&fuse_args) -> T>(options: &[&OsStr], f: F) -> T
     args.extend(options.iter().map(|s| CString::new(s.as_bytes()).unwrap()));
     let argptrs: Vec<_> = args.iter().map(|s| s.as_ptr()).collect();
     f(&fuse_args {
-        argc: argptrs.len() as i32,
+        argc: i32::try_from(argptrs.len()).unwrap(),
         argv: argptrs.as_ptr(),
```

Comment:
We should use expect() to provide more context. Otherwise it is hard to track the crash if it failed.

Reply:
Lint suggests to use unwrap_or_else